app/build.gradle

```diff
@@ -72,6 +72,9 @@ dependencies {
     compile "com.android.support:design:${SUPPORT_LIB_VERSION}"
     compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'com.squareup:otto:1.3.8'
+    compile('com.github.afollestad.material-dialogs:core:0.8.5.3@aar') {
```

Comment:
@vjs3 I believe this can be just done by using different styles. I don't think this is something extra in this library. We want limited number of dependencies for the app. Please think of it as if you can do this without using the library, why would you want to use it?